skopt/plots.py

```diff
@@ -118,15 +122,27 @@ def _format_scatter_plot_axes(ax, space, ylabel):
                 if j > 0:
                     ax_.set_yticklabels([])
                 else:
-                    ax_.set_ylabel("$X_{%i}$" % i)
+                    ax_.set_ylabel(dim_labels[i])
 
                 # for all rows except ...
                 if i < space.n_dims - 1:
                     ax_.set_xticklabels([])
                 # ... the bottom row
                 else:
                     [l.set_rotation(45) for l in ax_.get_xticklabels()]
-                    ax_.set_xlabel("$X_{%i}$" % j)
+                    ax_.set_xlabel(dim_labels[j])
+                
+                # configure plot for linear vs log-scale
+                priors = (space.dimensions[j].prior, space.dimensions[i].prior)
+                scale_setters = (ax_.set_xscale, ax_.set_yscale)
+                loc_setters = (ax_.xaxis.set_major_locator,
+                        ax_.yaxis.set_major_locator)
+                for set_major_locator, set_scale, prior in zip(loc_setters,
+                        scale_setters, priors):
```

Comment:
indent this so it lines up with `loc_setters` or make all three argument be on a new line?